src/v2_5/bid_request.rs

```diff
@@ -176,7 +177,7 @@ impl BidRequest {
             user: None,
             test: false,
             auction_type: AuctionType::FirstPrice,
-            tmax: 0,
+            tmax: Some(0),
```

Comment:
This should probably be `None` since a specified `tmax` of 0 would be impossible to satisfy.